.github/workflows/kickstart-tests.yml

```diff
@@ -0,0 +1,166 @@
+# Run kickstart tests in a PR triggered by a "/kickstart-test" comment from an organization member.
+name: kickstart-tests
+on:
+  issue_comment:
+    types: [created]
+
+permissions:
+  contents: read
+  checks: write
+
+jobs:
+  pr-info:
+    if: startsWith(github.event.comment.body, '/kickstart-test')
+    runs-on: ubuntu-latest
+    steps:
+      - name: Query comment author repository permissions
+        uses: octokit/request-action@v2.x
+        id: user_permission
+        with:
+          route: GET /repos/${{ github.repository }}/collaborators/${{ github.event.sender.login }}/permission
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      # restrict running of tests to users with admin or write permission for the repository
+      # see https://docs.github.com/en/free-pro-team@latest/rest/reference/repos#get-repository-permissions-for-a-user
+      # store output if user is allowed in allowed_user job output so it has to be checked in downstream job
+      - name: Check if user does have correct permissions
+        if: contains('admin write', fromJson(steps.user_permission.outputs.data).permission)
+        id: check_user_perm
+        run: |
+          echo "User '${{ github.event.sender.login }}' has permission '${{ fromJson(steps.user_permission.outputs.data).permission }}' allowed values: 'admin', 'write'"
+          echo "::set-output name=allowed_user::true"
+
+      - name: Get information for pull request
+        uses: octokit/request-action@v2.x
+        id: pr_api
+        with:
+          route: GET /repos/${{ github.repository }}/pulls/${{ github.event.issue.number }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+    outputs:
+      allowed_user: ${{ steps.check_user_perm.outputs.allowed_user }}
+      base_ref: ${{ fromJson(steps.pr_api.outputs.data).base.ref }}
+      sha: ${{ fromJson(steps.pr_api.outputs.data).head.sha }}
+      pr_number: ${{ github.event.issue.number }}
+
+  run:
+    needs: pr-info
+    if: needs.pr-info.outputs.allowed_user == 'true'
+    runs-on: [self-hosted, kstest]
+    timeout-minutes: 40
+    env:
+       STATUS_NAME: Kickstart test smoke tests
+       TARGET_BRANCH: ${{ needs.pr-info.outputs.base_ref }}
+       TEST_JOBS: 16
+       TEST_SELECTION: "--testtype smoke --skip-testtypes skip-on-fedora"
+       REPOSITORY: "http://dl.fedoraproject.org/pub/fedora/linux/development/rawhide"
```

Comment:
I think we should use something more stable.

Reply:
Right, what about https://dl.fedoraproject.org/pub/fedora/linux/releases/36 ?

Reply:
That should be better, we will have to update it once a year but at least there is a smaller chance that the test will fail because of problem in Fedora.

Reply:
Updated